src/vaadin-rich-text-editor.html

```diff
@@ -311,6 +312,7 @@
 
             /**
              * HTML representation of the rich text editor content.
+             * @type {string}
              */
             htmlValue: {
```

Comment:
There's no default value so should be `@type {string | undefined}`

Reply:
Removed.